app/src/pages/Home.svelte

```diff
@@ -0,0 +1,12 @@
+<script>
+  import Projects from '../components/Projects.svelte'
+</script>
+
+<style>
+	h1 {
+		color: purple;
+	}
```

Comment:
More indentation. Probably should add `prettify` so we don't have to think about this anymore.